### PR TITLE
PS-6910 Documented the new RocksDB variable: rocksdb_stats_level.

### PR DESCRIPTION
**Pull request by patrickbirch**
merged 5.7 version
fixed conflicts in source/myrocks/variables

### CODE REVIEW

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -1984,6 +1992,31 @@ to the info log.
 Default value is ``600``.
 Allowed range is up to ``2147483647``.
 
+.. variable:: rocksdb_stats_level
+
+  :version 5.7.30-33: Implemented
```
Comment (inikep) [inline doc/source/myrocks/variables.rst]: 8.0.20

File: doc/source/myrocks/variables.rst [hunk 2]
```diff
@@ -1984,6 +1992,31 @@ to the info log.
...
+  :scope: Global
+  :vartype: Numeric
+  :default: ``0``
+
+Controls the RocksDB statistics level. The default value is "0" (kExceptHistogramOrTimers),
+ which is the fastest level. The maximum value is "4".
+
+.. variable:: rocksdb_stats_recalc_rate
+
+  :version 5.7.23-23: Implemented
```
Comment (inikep) [inline doc/source/myrocks/variables.rst]: 8.0.x
This was not ported to 8.0 before?

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/myrocks/variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: Changed

File: doc/source/myrocks/variables.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: Removed 5.7 reference

**Review by inikep — APPROVED** [review]
LGTM

**Pull request merged**